client/src/features/action/assets/Asset.tsx

```diff
@@ -154,14 +154,14 @@ function Asset(): JSX.Element {
     });
 
   const aboutInfo = () => {
-    if (asset.organization !== null) {
```

Comment:
asset shouldn't have an organization
just put whatever in here since this will all probs get re-written in FE re-write initiative